src/admin/config.yml

```diff
@@ -143,6 +148,64 @@ collections:
         name: body
         widget: markdown
         i18n: true
+  - label: Project Subpages
+    label_singular: Project Subpage
+    name: project-subpages
+    icon: menu_open
+    i18n:
+      structure: multiple_folders
+    folder: src/collections/project-subpages/
+    extension: md
+    create: true
+    fields:
+      - label: Title
+        name: title
+        widget: string
+        i18n: true
+      - label: Language
+        name: lang
+        widget: hidden
+        default: "{{locale}}"
+      - label: UUID
+        name: uuid
+        widget: hidden
+        default: "{{uuid}}"
+        i18n: duplicate
+      - label: Slug
+        name: slug
+        widget: string
+        i18n: true
+        required: true
+        hint: Use this field to customize the URL of the project subpage. For example, if you want to display 'Regulating the Digital Domain reports' at `/projects/rtdd/reports/`, enter `reports`.
+      - label: Parent Project
+        name: parent
+        widget: relation
+        collection: projects
+        value_field: urlSlug
+        search_fields: [title]
+        display_fields: [title]
+        required: true
+        i18n: true
```

Comment:
This field should be duplicate; I was able to set a sub project under two different parent page for English and French.

Reply:
Actually, duplicate isn't what we want as that causes 2 duplicated sub pages to only appear in English page. I think it makes sense to leave it as is, but change the required field to false, as projects may have only English page; in that case, required would throw an error on save.

Reply:
The field is required; if there's no French translation the French translation can be disabled:

<img width="626" alt="Screenshot showing French translation being disabled from context menu" src="https://github.com/user-attachments/assets/1bb2acdd-a965-4e89-8962-4f5ac87c32fb" />